.github/workflows/native-build-development.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         run: |
           mvn -B clean install --fail-at-end -Pnative -Ddocker \
             -Dquarkus.native.container-build=true \
-            -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java8 \
+            -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:20.1.0-java11 \
```

Comment:
Should you switch to JDK 11 here too: https://github.com/quarkusio/quarkus-quickstarts/pull/624/files#diff-6fc556614702dfd6015c4f7b457f47c8R28 ?

Reply:
You are right, done